code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm

```diff
@@ -25,11 +36,17 @@
 
 	action_icon_state = "alien_evolve_praetorian"
 
+#define MAX_QUEEN_DEATHS 2
```

Comment:
Could you turn this into a xeno team variable, so admins are able to change it at runtime, instead of a hardcoded value?